tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -178,6 +196,15 @@ def montecarlo_main_loop(
     virt_packet_last_line_interaction_out_id = []
 
     for i in prange(len(output_nus)):
+        if show_progress_bar:
+            with objmode:
+                update_packet_pbar(
+                    1,
```

Comment:
Even if this 1 should be hard coded, it would be better to define it as a variable with a descriptive name so that future developers know what it does.